libs/docs/src/docs/2.2.0/create-devfiles-with-templates.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Creating devfiles with templates
+description: Creating devfiles with templates
+---
+
+Most dev tools can utilize the devfile registry to 
+fetch the stacks needed to begin development on projects. 
+Sometimes, however, you might need to create a devfile 
+from scratch or to create your own devfile template(s). 
+This guide runs through the process, starting from a minimum devfile and 
+building sample templates for common use cases.
+
+## Prerequisites
+
+- [Versions](./versions)
+- [Metadata](./metadata)
+- [Adding components](./adding-components)
+- [Adding commands](./adding-commands)
+
```

Comment:
Since the procedure goes through adding a starter project, we should also add a link to the topic [Adding projects](./adding-projects)

Reply:
Added [Defining starter projects](https://devfile.io/docs/2.2.0/defining-starter-projects).